bigbio/hub/hub_repos/cardiode/README.md

```diff
@@ -0,0 +1,43 @@
+---
+language:
+  - ger [This needs to be a supported huggingface language code]
```

Comment:
I don't think we want the boilerplate in square brackets here

Reply:
also, lets check if this is a supported code. "de" is used here https://github.com/bigscience-workshop/biomedical/pull/866/files

Reply:
looks like HF accepts 2 or 3 letter codes, so I'll merge this